### ISSUE TITLE
Add a sample spatial files + update overview section 

### ISSUE THREAD

**Issue by MCatherine1994** [bcgov/nr-fom#635]
* Update the wording in the format overview section as well, Julius will send us a new word document.
* Update current spatial submission overview to be clear format (UI design)
* On FOM Admin site, when user submit spatial file, add a sample GeoJSON template to allow them download, fill it out and upload to FOM. Currently Julius need to send the example file to the users, it will be better if the user can download directly. 

**Acceptance Criteria**
- [x] Allow download of sample spatial files (geojson) 
- [x] The "download template" link should give a GeoJSON file based on the object type. When a user selects "cutblock" provide a GeoJSON that is for cutblocks", when a user selects "retention area", provide GeoJSON file relevant to retention are
- [x] Minor wording update for sample spatial files button label from "Download Templates" to "Download Sample Spatial Files".
- [x] Rewrite the spatial overview with FAQ style.
- [x] Allow download for all spatial overview FAQ information as PDF. 

**Additional context**
- ![Image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/2e5ad641-a1b5-495c-8d49-4ff49c581e92)

New file, update page content with this file: 
*content approved by Sean Musie and Aaren Ritchie-Bonar

[FOM overview (FAQ) Jul 4, 2024 v2.docx](https://github.com/user-attachments/files/16103225/FOM.overview.FAQ.Jul.4.2024.v2.docx)

follow this format: 

![Image](https://github.com/bcgov/nr-fom/assets/121979429/ec0146b6-4a16-42a0-8d74-46c0b25ba9fd)

Note that this applies to cutblocks and road sections - but not retention areas as they don't have a development date.
Validation just on backend is fine

**Comment by MCatherine1994** [comment]
From Julius:

[JSON_Format Overview_Final_08.04.24.docx](https://github.com/bcgov/nr-fom/files/15489654/JSON_Format.Overview_Final_08.04.24.docx)

**Comment by OlgaLiber2** [comment]
@MCatherine1994 @ianliuwk1019 Sean approved. Please use this mockup. By default files should be saved as json.



![Image](https://github.com/bcgov/nr-fom/assets/121979429/f06efe77-e661-41a1-b043-b4e9acdd2cf4)

**Comment by MCatherine1994** [comment]
Hi Olga @OlgaLiber2, are we still doing the Q/A sections? or this one is mainly just for adding a download dropdown?

**Comment by MCatherine1994** [comment]
Create the sample spatial files, contents are copied from the word documentation:

[sample_wtra_file.json](https://github.com/user-attachments/files/16075337/sample_wtra_file.json)
[sample_road_section_file.json](https://github.com/user-attachments/files/16075338/sample_road_section_file.json)
[sample_cut_block_file.json](https://github.com/user-attachments/files/16075336/sample_cut_block_file.json)

**Comment by OlgaLiber2** [comment]
content for the FAQ has been approved by Sean and Aaren. I updated the file in this ticket. @MCatherine1994 @ianliuwk1019 

Outstanding: design a download button to download the FAQ as a pdf.

**Comment by ianliuwk1019** [comment]
Hi @OlgaLiber2 Catherine already adds the download "dropdown" for spatial files. Does the design change for this again (from screenshot)?

**Comment by OlgaLiber2** [comment]
@ianliuwk1019 Design is the same, still dropdowns. They asked for a button to be able to download the whole thing. So like a download button to save a pdf file.

**Comment by ianliuwk1019** [comment]
Hi @OlgaLiber2 , since we previously just added the spatial files download button, I assume this section in the document isn't needed anymore:

![Image](https://github.com/bcgov/nr-fom/assets/81595625/1f905f2a-2a0b-4241-82b5-a604ba56e4bd)

![Image](https://github.com/bcgov/nr-fom/assets/81595625/de53f41f-840e-4efd-9123-b0ca2522dd16)

**Comment by OlgaLiber2** [comment]
@ianliuwk1019 let's keep it out for now :)

**Comment by Kghiya** [comment]
[Download PDF](https://app.mural.co/t/climatechange4363/m/climatechange4363/1718909618235/2998998ec160311cb0c225455486afee685c73e0?wid=0-1720723394304) button design for the FAQs section
@ianliuwk1019

**Comment by Kghiya** [comment]
[FAQs for submitting geospatial files.pdf](https://github.com/user-attachments/files/16182298/FAQs.for.submitting.geospatial.files.pdf)

Here is the PDF to be attached to the button.

**Comment by ianliuwk1019** [comment]
Hi @OlgaLiber2 pdf download added, you can have a look again at: https://fom-7.apps.silver.devops.gov.bc.ca/admin.
I kept the pdf download button the same as others to be consistent.

**Comment by ianliuwk1019** [comment]
Basil intends to review the doc content again and provide better wording if necessary.

**Comment by ianliuwk1019** [comment]
Hi @Kghiya , Basil has an updated overview document: [FOM.overview.FAQ.Jul.4.2024.v2.docx](https://github.com/user-attachments/files/16446838/FOM.overview.FAQ.Jul.4.2024.v2.docx) .
Would you help to create a pdf for download version?
Thank you!!

.Final version: [FOM overview (FAQ) Jul 12 2024 - Basil_JH.docx](https://github.com/user-attachments/files/16461002/FOM.overview.FAQ.Jul.12.2024.-.Basil_JH.docx)

**Comment by Kghiya** [comment]
@OlgaLiber2 I have the updated document above and working on PDF. I will update the content but making sure that we are still not adding the example files. right?

**Comment by OlgaLiber2** [comment]
@Kghiya yes, we're not adding it in the pdf and @ianliuwk1019 you can take that out of the overview section as well because we have the templates at the top of the page.

**Comment by Kghiya** [comment]
Updated PDF 

[FAQs for submitting geospatial files.pdf](https://github.com/user-attachments/files/16473933/FAQs.for.submitting.geospatial.files.pdf)

@ianliuwk1019

**Comment by ianliuwk1019** [comment]
Hi @Kghiya ,
Could you check the pdf you attached is the correct one?
I compare the content between .doc and .pdf and they are different.
The final pdf should be based on "FOM overview (FAQ) Jul 12 2024 - Basil_JH.docx".
Thanks again!

**Comment by Kghiya** [comment]
Thank you Ian for calling that out. Updated PDF is here

[FAQs for submitting geospatial files.pdf](https://github.com/user-attachments/files/16478117/FAQs.for.submitting.geospatial.files.pdf)

**Comment by ianliuwk1019** [comment]
Thank you @Kghiya !!

Hi @Kghiya ,
The following section has some differences, could you check/modify again?
![Image](https://github.com/user-attachments/assets/d3b18c4b-95a9-4f0e-9be3-fc16cd19b29a)

**Comment by Kghiya** [comment]
@ianliuwk1019 

[FAQs for submitting geospatial files.pdf](https://github.com/user-attachments/files/16532461/FAQs.for.submitting.geospatial.files.pdf)

Thank you!

**Comment by ianliuwk1019** [comment]
Hi @OlgaLiber2 this is deployed to test. Thank you so much for reviewing this!!
